web/src/enterprise/campaigns/detail/changesets/ExternalChangesetNode.tsx

```diff
@@ -57,31 +57,31 @@ export const ExternalChangesetNode: React.FunctionComponent<ExternalChangesetNod
     location,
     extensionInfo,
 }) => {
-    const ReviewStateIcon = changesetReviewStateIcons[node.reviewState]
+    const ReviewStateIcon = node.reviewState && changesetReviewStateIcons[node.reviewState]
     const ChangesetCheckStateIcon = node.checkState
         ? changesetCheckStateIcons[node.checkState]
         : changesetCheckStateIcons[ChangesetCheckState.PENDING]
-    const changesetState = node.state
+    const changesetState = node.externalState
 
     const changesetNodeRow = (
         <div className="d-flex align-items-start m-1 ml-2">
             <div className="changeset-node__content flex-fill">
                 <div className="d-flex flex-column">
                     <div className="m-0 mb-2">
                         <h3 className="m-0 d-inline">
-                            <ChangesetStateIcon state={changesetState} />
+                            <ChangesetStateIcon externalState={changesetState || ChangesetExternalState.OPEN} />
```

Comment:
I'd also rename these, the sooner the better (doesn't mean in this PR):

```suggestion
                            <ChangesetExternalStateIcon externalState={changesetExternalState || ChangesetExternalState.OPEN} />
```

Reply:
I'm going to rip those out in a follow-up, to keep the diff small-ish. There were a lot of places that I didn't want to touch yet